arbitrary-file-write/file-delete/README.md

```diff
@@ -8,4 +8,14 @@
 * `rm /etc/group`
 	* system broken. tested: `sudo, login, su`
 * `rm /etc/sudoers`
-	* only `sudo` broken (also tested empty file)
\ No newline at end of file
+	* only `sudo` broken (also tested empty file)
+* `rm /etc/fstab`
+    * say goodbye to your system ig. most likely it'll boot to some rescue mode where you'll have to log in over a display or serial. 
+
+### shell profile fallbacks
+See shell profile section in file-create, deleting one file will cause the shell to look for another file. Use case: the system sets the path in one of the shell profiles so it uses an up to date version of some programming language, but we bypass that by deleting the profile. 
+
+### iptables neturalized (untested)
+Some systems (TODO: test which systems do), persist iptables to a specific file so that rules can be reapplied on reboot. Theoritically one could delete such a file so that no rules are applied on reboot. 
```

Comment:
really good idea! definitely worth looking into. But maybe we should move these ideas into a discussion, TODO file or issue?